### PR TITLE
VDS-2473 Return error info array instead of erros codes for pswd validation query

### PR DESCRIPTION
**Pull request by trueboroda**

### CODE REVIEW

File: src/VirtoCommerce.ExperienceApiModule.Core/Queries/PasswordValidationQueryHandler.cs [hunk 1]
```diff
@@ -32,9 +34,19 @@ public virtual async Task<PasswordValidationResponse> Handle(PasswordValidationQ
             foreach (var passwordValidator in userManager.PasswordValidators)
             {
                 var validationResult = await passwordValidator.ValidateAsync(userManager, null, request.Password);
-
                 result.Succeeded &= validationResult.Succeeded;
-                errorCodes.AddRange(validationResult.Errors.Select(x => x.Code));
+
+                result.Errors.AddRange(validationResult.Errors.Select(x =>
```
Comment (Vectorfield4) [inline src/VirtoCommerce.ExperienceApiModule.Core/Queries/PasswordValidationQueryHandler.cs:39]: Move function from the selector function to the protected virtual method.

File: src/VirtoCommerce.ExperienceApiModule.Core/Queries/PasswordValidationResponse.cs [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
+using VirtoCommerce.ExperienceApiModule.Core.Models;
 
 namespace VirtoCommerce.ExperienceApiModule.Core.Queries
 {
     public class PasswordValidationResponse
     {
         public bool Succeeded { get; set; }
 
-        public IList<string> ErrorCodes { get; set; }
+        public IList<IdentityErrorInfo> Errors { get; set; }
```
Comment (Vectorfield4) [inline src/VirtoCommerce.ExperienceApiModule.Core/Queries/PasswordValidationResponse.cs:10]: Avoid breaking changes.
```suggestion
        [Obsolete("User Errors insted")]
        public IList<string> ErrorCodes { get; set; }
        public IList<IdentityErrorInfo> Errors { get; set; }
```

File: src/VirtoCommerce.ExperienceApiModule.Core/Schemas/PasswordValidationType.cs [hunk 3]
```diff
@@ -1,14 +1,28 @@
 using GraphQL.Types;
+using VirtoCommerce.ExperienceApiModule.Core.Models;
 using VirtoCommerce.ExperienceApiModule.Core.Queries;
 
 namespace VirtoCommerce.ExperienceApiModule.Core.Schemas
 {
+    public class PasswordValidationErrorType : ObjectGraphType<IdentityErrorInfo>
```
Comment (Vectorfield4) [inline src/VirtoCommerce.ExperienceApiModule.Core/Schemas/PasswordValidationType.cs:7]: Move to `PasswordValidationErrorType.cs`

**Review by Vectorfield4 — CHANGES_REQUESTED** [review]

File: src/VirtoCommerce.ExperienceApiModule.Core/Queries/PasswordValidationQueryHandler.cs (hunk 1 above)
Comment (trueboroda) [reply src/VirtoCommerce.ExperienceApiModule.Core/Queries/PasswordValidationQueryHandler.cs:39]: Why?

File: src/VirtoCommerce.ExperienceApiModule.Core/Queries/PasswordValidationResponse.cs (hunk 2 above)
Comment (Vectorfield4) [reply src/VirtoCommerce.ExperienceApiModule.Core/Queries/PasswordValidationResponse.cs:10]: Not a breaking change.

**Review by Vectorfield4 — APPROVED** [review]

**Pull request merged**